contrib/babelfishpg_tsql/src/hooks.c

```diff
@@ -275,6 +278,9 @@ InstallExtendedHooks(void)
 	planner_hook = pltsql_planner_hook;
 	prev_transform_check_constraint_expr_hook = transform_check_constraint_expr_hook;
 	transform_check_constraint_expr_hook = transform_like_in_add_constraint;
+
+	prev_verify_valid_scale_in_variable_length_datatypes_hook = verify_valid_scale_in_variable_length_datatypes_hook;
+	verify_valid_scale_in_variable_length_datatypes_hook = pltsql_verify_valid_scale_in_variable_length_datatypes;
```

Comment:
Does `validate_var_datatype_scale_hook` makes sense? (Trying to keep the name short)

Reply:
done